sdks/java/extensions/protobuf/src/test/java/org/apache/beam/sdk/extensions/protobuf/ProtoMessageSchemaTest.java

```diff
@@ -1,249 +0,0 @@
-/*
```

Comment:
Do NOT delete

Reply:
Yep if I undo the delete of ProtoMessageSchema I won't delete this one.